1907-05-27.xml

```diff
@@ -0,0 +1,1512 @@
+                            Steamers "VICTORIA," "PURITAN," &amp; "CERMANIA."<lb/> Regular Weekly Departures to
+                            the SECOND CATARACT by the S.S. "NUBIA."<lb/> THROUGH BOOKINGS TO KHARTOUM,
+                            GONDOKORO AND THE WHITE NILE.<lb/> Steamers and Dahabeahs for private charter.
+                            Steam Tugs and Steam Launches for hire.<lb/> FREIGHT SERVICE BY STEAM BARGES
+                            BETWEEN CAIRO AND ALEXANDRIA.<lb/> Working in conjunction and under special
+                            arrangement with the<lb/> “Upper Egypt Hotels Company."</p>
+                        <p>For details and programmes apply to HEAD OFFICE: Sharia Boulac, "Grand Continental 
+                            Hotel Buildings.” CAIRO. <measure
+                                type="indexNo">31-3-07</measure></p>
+                        <p>Hamburg-Amerika Linie.</p>
+                        <table>
+                            <row>
+                                <cell> </cell>
+                                <cell>Leaves Port Said</cell>
+                                <cell>Arrives Naples</cell>
+                                <cell>Arrives Hamburg</cell>
+                            </row>
+                            <row>
+                                <cell>S.S. Habsburg</cell>
+                                <cell><date when="1907-05-04"></date>May 4th </cell>
+                                <cell><date when="1907-05-08"></date>May 8th</cell>
+                                <cell><date when="1907-05-20"></date>May 20th </cell>
+                            </row>
+                            <row>
+                                <cell>S.S. Rheania</cell>
+                                <cell><date when="1907-06-15">June 15th</date></cell>
+                                <cell><date when="1907-06-19">June 19th</date></cell>
+                                <cell><date when="1907-07-01">July 1st</date></cell>
+                            </row>
+                            <row>
+                                <cell>S.S. Hohenstaufen</cell>
+                                <cell><date when="1907-07-13">June 13th</date></cell>
+                                <cell><date when="1907-07-17">July 17th</date></cell>
+                                <cell><date when="1907-07-29">July 29th</date></cell>
+                            </row>
+                            <row>
+                                <cell>S.S. Silesia</cell>
+                                <cell><date when="1907-08-10">August 10th</date></cell>
+                                <cell><date when="1907-08-14">August 14th</date></cell>
+                                <cell><date when="1907-08-26">August 26th</date></cell>
+                            </row>
+                        </table>
+                        <p>These steamers call also at Plymouth if needed.</p>
+                        <
```

Comment:
don't end the section here--you don't need two </div> tags. This closing tag should come just before the end of the page